src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,154 @@
+    transitionTime: TlvField(2, TlvNullable(TlvUInt16)),
+    optionsMask: TlvField(3, TlvUInt8),
+    optionsOverride: TlvField(4, TlvUInt8),
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.4 */
+const  StopCommand = TlvObject({
+    optionsMask: TlvField(0, TlvUInt8),
+    optionsOverride: TlvField(1, TlvUInt8),
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.5  */
+const MoveToClosestFrequencyCommand = TlvObject({
+    frequency: TlvField(0,TlvUInt16),
+})
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.4 */
+const features = {
+    /** Dependency with the On/Off cluster */
+    OnOff: BitFlag(0),
+    /** Behavior that supports lighting applications */
+    Lighting: BitFlag(1),
+    /** Supports frequency attributes and behavior. */
+    Frequency: BitFlag(2)
+};
+
+ /** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.5 */
+const attributes = {
+    /** the current level of this device */
+    currentLevel: Attribute(0x0,  TlvUInt8, {default: 0}), 
+
+    /** time until the current command is complete in 1/10ths of sec */
+    remainingTime: Attribute(0x01, TlvUInt16, {default: 0}),
+
+    /** the minimum value of CurrentLevel  */
+    minLevel: OptionalAttribute(0x02, TlvUInt8),
+
+    /** the maximum value of CurrentLevel */
+    maxLevel: OptionalAttribute(0x03, TlvUInt8, {default: 254}),
+
+    /** the frequency at which the device is at CurrentLevel */
+    currentFrequency: Attribute(0x04, TlvUInt16, {default: 0}),
+
+    /** min value of CurrentFrequency capable of being assigned */
+    minFrequency: Attribute(0x05, TlvUInt16, {default: 0}),
+
+    /** max value of CurrentFrequency capable of being assigned */
+    maxFrequency: Attribute(0x06, TlvUInt16, {default: 0}),
+
+    /** time to move to/from the target when On/Off received by On/Off cluster on the same endpoint */
+    onOffTransitionTime: OptionalAttribute(0x10, TlvUInt16, {default: 0}),
+
+    /** CurrentLevel when OnOff attr of On/Off cluster on the same endpoint is TRUE */
+    onLevel: Attribute(0x11,  TlvNullable(TlvUInt8)),
+
+    /** time taken to move the current level from the min level to the max level */
+    onTransitionTime: OptionalAttribute(0x12, TlvNullable(TlvUInt16,)),
+
+    /** time taken to move the current level from the max level to the minlevel  */
+    offTransitionTime: OptionalAttribute(0x13, TlvNullable(TlvUInt16),),
+
+    /** default rate in units per second when a Move command has a null Rate */
+    defaultMoveRate: OptionalAttribute(0x14, TlvNullable(TlvUInt8),),
+
+    options: Attribute(0x0f,  TlvUInt8, {default: 0 }),
+    startUpCurrentLevel: Attribute(0x4000, TlvNullable(TlvUInt8),),  
+};
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.6 */
+const commands = {
+    /** Move from the current level to the specified level */
+    moveToLevel: OptionalCommand(0x0, MoveToLevelCommand, 0x0, TlvNoResponse), // FIXME - all need resposne?
```

Comment:
No response is correct that way. Response Y means that a status OK is send out automatically but no specific response